sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.1.1 (2020-01)
```

Comment:
Your package version is 12.0.1. Should this be, too?

Reply:
Fixed.